pkg/integrations/v2/eventhandler/integration.go

```diff
@@ -38,7 +39,8 @@ type Config struct {
 	// If you would like to limit events to a given namespace, use this parameter.
 	Namespace string `yaml:"namespace,omitempty"`
 	// Extra labels to append to log lines
-	ExtraLabels labels.Labels `yaml:"extra_labels,omitempty"`
+	ExtraLabels labels.Labels        `yaml:"extra_labels,omitempty"`
+	Common      common.MetricsConfig `yaml:",inline"`
```

Comment:
I don't think embedding all of MetricsConfig is appropriate here because eventhandler is a logging integration. Can we just manually copy the instance key field instead of bringing in everything else?